tests/test_multi_ifdown.py

```diff
@@ -102,11 +103,10 @@ def test_005_set_candidate_priorities():
     assert node2.get_replication_quorum()
 
     # also let's see synchronous_standby_names here
-    print("Monitor: %s" % node1.get_synchronous_standby_names())
-    print(
-        "Node 1:  %s"
-        % node1.run_sql_query("show synchronous_standby_names")[0][0]
-    )
+    # remember to sort by candidate priority then name
+    ssn = "ANY 1 (pgautofailover_standby_3, pgautofailover_standby_2)"
```

Comment:
Maybe we should remove this special case ordering. Postgres doesn't do anything special. (for another PR)

Reply:
It's nice for the unit tests, so there's that... otherwise it's not useful, but also I think it's neutral, as Postgres will just not care about the ordering when using `ANY` anyway...